app.css

```diff
@@ -152,6 +154,10 @@ body {
     display: none;
   }
 
+  .component-container > .grid {
+    grid-template-columns: minmax(0, 1fr);
+  }
+
   .link-to-top {
```

Comment:
when to use unset , can you elaoborate on this!!

Reply:
yeah, sure. Actually initially `link-to-top` has a `display: none` class, which I have set in `docs.css`. And when the screen width is coming down to `63em`, I am unsetting that display. Basically whatever it has default `display` value, just saying to apply that.

And when the screen width comes down to `44em`, I am again setting it to `display: none`. It is for responsiveness. 

If you want to read more about this, you can refer this, [unset mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/unset)